spring-cloud-gcp-samples/spring-cloud-gcp-secretmanager-sample/src/main/resources/bootstrap.properties

```diff
@@ -1,15 +1,6 @@
-# You can directly load the secret into a variable like this example
-# This demonstrates multiple ways of loading the same application secret using template syntax.
-#
-# Please refer to the documentation for the protocol syntax.
-#
-# Examples:
-# my-app-secret-3=${sm://projects/<YOUR_PROJECT_ID>/secrets/application-secret}
-# my-app-secret-4=${sm://projects/<YOUR_PROJECT_ID>/secrets/application-secret/version/latest}
-# my-app-secret-5=${sm://<YOUR_PROJECT_ID>/application-secret/latest}
+# Configuration-level settings for Secret Manager must be specified in a bootstrap.properties file because
+# Secret Manager secrets are loaded in the application by Spring Bootstrap Context
```

Comment:
I don't think the rationale is very clear - perhaps "in a bootstrap.properties file because Secret Manager secrets must be fetched/resolved before the application.properties files are resolved" ?

Reply:
Done.